Header header
float32 V1
float32 V2
float32 V3
float32 V4
float32 C1
float32 C2
float32 C3
float32 C4
float32 THRUST1
float32 STRAIN1
float32 THRUST2
float32 STRAIN2
float32 THRUST3
float32 STRAIN3
float32 THRUST4
float32 STRAIN4
uint16 RPM1
uint16 RPM2
uint16 RPM3
uint16 RPM4
